core/services/fluxmonitor/flux_monitor_test.go

```diff
@@ -42,6 +42,35 @@ func ensureAccount(t *testing.T, store *store.Store) common.Address {
 	return acct.Address
 }
 
+func TestConcreteFluxMonitor_Start_withEthereumDisabled(t *testing.T) {
+	tests := []struct {
+		name        string
+		enabled     bool
+		wantStarted bool
+	}{
+		{"enabled", true, false},
+		{"disabled", false, true},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			config, cleanup := cltest.NewConfig(t)
+			defer cleanup()
+			config.Config.Set("ETH_DISABLED", test.enabled)
+			store, cleanup := cltest.NewStoreWithConfig(config)
+			defer cleanup()
+			runManager := new(mocks.RunManager)
+
+			fm := fluxmonitor.New(store, runManager)
+			logBroadcaster := fm.(fluxmonitor.MockableLogBroadcaster).MockLogBroadcaster()
+
+			err := fm.Start()
```

Comment:
Where do we stop it?

Reply:
Good call, had to change the stop function.

Reply:
If `fm.Start` errors and we call `fm.Stop` is this a problem?

Reply:
Depends how it errors, but I think it will probably hang.

Reply:
Should we check if error is not nil before calling stop?

Reply:
Added a requirement there.